backend/app/api/v1/endpoints/telco/telcoJobs.py

```diff
@@ -82,7 +82,7 @@ async def jobs(
     "/api/v1/telco/filters",
     summary="Returns data to build the filter",
     description="Returns a list of jobs in the specified dates of requested size. \
-            If not dates are provided the API will default the values. \
+            If not dates are provided the API will use the following values as defaults. \
```

Comment:
"If dates are not provided"

Reply:
It may also be clearer if it ended in a colon `:`.